src/libraries/System.Data.OleDb/ref/System.Data.OleDb.cs

```diff
@@ -180,9 +182,9 @@ public sealed partial class OleDbDataReader : System.Data.Common.DbDataReader
         public override void Close() { }
         public override bool GetBoolean(int ordinal) { throw null; }
         public override byte GetByte(int ordinal) { throw null; }
-        public override long GetBytes(int ordinal, long dataIndex, byte[] buffer, int bufferIndex, int length) { throw null; }
+        public override long GetBytes(int ordinal, long dataIndex, byte[]? buffer, int bufferIndex, int length) { throw null; }
         public override char GetChar(int ordinal) { throw null; }
-        public override long GetChars(int ordinal, long dataIndex, char[] buffer, int bufferIndex, int length) { throw null; }
+        public override long GetChars(int ordinal, long dataIndex, char[]? buffer, int bufferIndex, int length) { throw null; }
```

Comment:
is null buffer actually allowed or is it only for zero length?

Reply:
A null buffer makes the method just return the number of characters in the column, without returning it in any way ([this is even in the docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.data.sqlclient.sqldatareader.getchars?view=sqlclient-dotnet-core-2.0#Microsoft_Data_SqlClient_SqlDataReader_GetChars_System_Int32_System_Int64_System_Char___System_Int32_System_Int32_)...).